libsolutil/Whiskers.h

```diff
@@ -73,7 +73,8 @@ class Whiskers
 	using StringMap = std::map<std::string, std::string>;
 	using StringListMap = std::map<std::string, std::vector<StringMap>>;
 
-	explicit Whiskers(std::string _template);
+	explicit Whiskers(std::string _template = "");
+	void setTemplate(std::string _template);
```

Comment:
Why not just use `operator=`?

Reply:
True. I removed `setTemplate`, but will leave the default initialisation in the constructor.